### PR TITLE
Fixed Coming Soon: Closes #409

### LINKED ISSUE #409: [Enhancements]
### What would you like to share or ask?

I was exploring the website and I would like to suggest some trivial changes - which I feel will improve the overall UI/UX 

1.  Alignment of items - aligning the cards to the center will be a better visual experience , the space on the right creates an unnecessary void

image

2.  Github, twitter , etc. can be opened up on a new page - As of now, on clicking any of the links, they load in the site itself. Creating a new window for them will be better.

3.  Not found message - when a search query does not exist, a message regarding the same can be displayed

image

4.  For topics which do not have any content as of now, the "coming soon" message is displayed. I think it will be better to display the message in the center of the frame rather than the corner, and also preferably in a bigger font and some animations - like the one which appears when one reaches the bottom of the screen. Also, making the entire text flicker instead of just the first word will be prettier, but let me know what you think :)
 

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by JA-26-01**
## Changes proposed

Issue 2 in of #409 

It modifies the coming soon message which is displayed when there is no content to be shown. The message has been modifed to "Be the first to add by creating a github issue **here**", with a link to create and contribute to the same. The text has also been shifted to the centre of the screen for better visibility and user experience. 

## Screenshots

Before:

image

After proposed changes:

image

### CODE REVIEW

**Review by rupali-codes — CHANGES_REQUESTED** [review]
Thanks, i've 2 suggestions
1. Open link in new tab
2. Update `ComingSoon` file name

File: components/ComingSoon/ComingSoon.tsx [hunk 1]
```diff
@@ -3,11 +3,11 @@ import { FC } from "react";
 const ComingSoon:FC<{}> = () => {
 
   return (
-    <p className="pt-6 md:p-0">
+    <p className="pt-6 md:p-0 coming-soon">
+      <span className="text-2xl font-semibold">Be the first to add by creating a GitHub issue</span>{" "}
```
Comment (rupali-codes) [inline components/ComingSoon/ComingSoon.tsx:7]:
```suggestion
      <span className="text-2xl">Be the first to add by creating a GitHub issue</span>{" "}
```

File: components/ComingSoon/ComingSoon.tsx [hunk 2]
```diff
@@ -3,11 +3,11 @@ import { FC } from "react";
...
       <span className="animate-pulse text-2xl font-semibold text-purple-500">
```
Comment (rupali-codes) [inline components/ComingSoon/ComingSoon.tsx:8]:
```suggestion
      <span className="animate-pulse text-2xl text-purple-500">
```

**Comment by JA-26-01** [comment]
@rupali-codes I have included the suggestions as well as the reviews you made. You can go through it once.

**Review by rupali-codes — APPROVED** [review]
Looks good to me, thanks

**Pull request merged**